src/pages/List.vue

```diff
@@ -6,12 +6,18 @@
 
 <script>
 import MonitorList from "../components/MonitorList.vue";
+import mobile from "../mixins/mobile";
 
 export default {
     components: {
         MonitorList,
     },
 };
+
```

Comment:
I don't know what the guarantees are when vue would run this code.
I think it would run during the init options api, but not when the browser changes its width => f.ex. gets larger

Docs:
https://vuejs.org/guide/essentials/lifecycle.html